pkg/metricsstore/metricsstore.go

```diff
@@ -134,6 +137,13 @@ func init() {
 			"success",       // further labels if the operation succeeded or not
 		})
 
+	defaultMetricsStore.ProxyBroadcastEventCount = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: metricsRootNamespace,
+		Subsystem: "proxy",
+		Name:      "broadcast_event_count",
+		Help:      "represents the number of ProxyBroadcast events published by the OSM controller",
```

Comment:
For consistency, it looks like the `Help` string should start with a capital letter like the others.

Reply:
+1 though a nit.

Reply:
Good catch. Shalier made a change to make all `Help` strings begin with a capital letter.